### PR TITLE
add openshift clusters dashboard with ability to switch between clusters

### LINKED ISSUE #34: Grafana dashboard error
Error by dynamic provisioner 
```
lvl=eror msg="failed to load dashboard from " logger=provisioning.dashboard type=file name=federated-prometheus file=/etc/grafana/provisioning/dashboards/openshift-cluster-monitoring.json error="Dashboard title cannot be empty"
```

### PR DESCRIPTION
**Pull request by charlesakalugwu**
closes #34 and #42

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /env
 /src/go/azure-purge
 /src/go/prometheus-sidecar
+/.idea
```
Comment (thanasisk) [inline .gitignore]: please adjust your local git settings and do not modify the .gitignore - else we have to add every extension depending on engineer's editor

Comment (charlesakalugwu) [reply .gitignore]: the .idea is ignored in our other projects :)

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-monitoring.json [hunk 2]
```diff
@@ -1,5138 +1,5762 @@
...
-            "rangeMaps": [
-              {
-                "from": "null",
-                "text": "N/A",
-                "to": "null"
-              }
-            ],
-            "span": 2,
-            "sparkline": {
-
```
Comment (kwoodson) [inline sre/grafana-deployment/provisioning/dashboards/openshift-cluster-monitoring.json]: Add newline char.

File: .gitignore (hunk 1 above)
Comment (thanasisk) [reply .gitignore]: if that is the case, then ignore my previous comment

**Pull request merged**